### DISCUSSION TITLE
Dependency Lookup Warnings

### DISCUSSION THREAD

**Discussion by jsaveker** [onedr0p/cluster-template#971] [Q&A] [answered]
I recently started to get the following warning in renovate dashboard:

Renovate failed to look up the following dependencies: Failed to look up galaxy-collection package ansible.posix, Failed to look up galaxy-collection package ansible.utils, Failed to look up galaxy-collection package community.general, Failed to look up galaxy-collection package community.sops, Failed to look up galaxy-collection package kubernetes.core.

I noticed that certain PRs are failing to pass checks such as:

[feat(container)!: Update image app-template to v2]

It appears to be related to bsk-w helm charts.

Any pointers on how I may troubleshoot/resolve this?

**Comment by onedr0p** [comment, marked as answer]
See this upstream issue, nothing to do here but wait until Renovate team fixes it.

https://github.com/renovatebot/renovate/issues/24981

**Reply by jsaveker** [reply]
Awesome... thank you for the super fast reply!
